### ISSUE TITLE
Does not work with windows

### ISSUE THREAD

**Issue by hfdxlh** [Pakisan/jasyncapi-idea-plugin#38]
```<script>
            AsyncApiStandalone.render({
                schema: {
                    url: 'http://localhost:63342/asyncapi/resources?schemaUrl=C:\Users\uisee\AppData\Local\Temp\jasyncapi-idea-plugin-1652714434031.yaml',
                    options: { method: "GET", mode: "cors" },
                },
                config: {
                    show: {
                        sidebar: true,
                    }
                },
            }, document.getElementById('asyncapi'));
        </script>


`Uncaught SyntaxError: Invalid Unicode escape sequence`

**Comment by Pakisan** [comment]
@hfdxlh hi.

Can you check it in new release?

[jasyncapi-idea-plugin-1.7.1+idea2021.zip](https://github.com/Pakisan/jasyncapi-idea-plugin/files/9793208/jasyncapi-idea-plugin-1.7.1%2Bidea2021.zip)

**Comment by Pakisan** [comment]
Fixed in [1.7.1](https://github.com/Pakisan/jasyncapi-idea-plugin/releases/tag/1.7.1%2Bidea2021). Will be published to plugins within 2 days, after moderation.
